### PR TITLE
Add current user notifications

### PR DESCRIPTION
**Pull request by medson10**

### CODE REVIEW

**Review by georgelima — APPROVED** [review]

File: executable/SummarizeApp.re [hunk 1]
```diff
@@ -10,55 +10,298 @@ let mockedSummary: User.summary = {
...
+            ~args=Arg.[],
+            ~resolve=(info, p: edge(User.Message.t)) =>
+            p.node
+          ),
+        ]
+      )
+    )
+  );
+
+let messagesConnection =
```
Comment (fakenickels) [inline executable/SummarizeApp.re:200]: nice we can probably mount a helper after to help boot the connection structure

File: schema/User.re [hunk 2]
```diff
@@ -1,14 +1,36 @@
+module Summary = {
+  type t = {
+    unread: int,
+    total: int,
+    projects: int,
+    organizations: int,
+  };
+};
+
+module Message = {
```
Comment (fakenickels) [inline schema/User.re:10]: nice, when we introduce the db each entity here will be on its own file
`Message.re` `Notification.re`

Comment (medson10) [reply schema/User.re:10]: top

File: executable/SummarizeApp.re (hunk 1 above)
Comment (medson10) [reply executable/SummarizeApp.re:200]: I just had an idea here, gonna push soon

**Pull request merged**